### PR TITLE
Feature/fitting clothing topic

### LINKED ISSUE #517: [ FEATURE REQUEST ] - Natsuki struggling to find fitting clothes
# Overview:
While drawing sleeves for #463 I noticed some sleeves are really big, which lead to the idea of a topic for Natsuki!
Natsuki could complain about struggling to find fitting clothes as result of her petite body. With higher affinity it can be revelead, that she actually started to shop in the kid section, partly because they are cheaper, but also because they are simply smaller. Natsuki probably added her own style to this outfits by stitching, for example, to cover up the might childish design. We could even make the `pastel_goth_outfit` unlockable after viewing this topic.

## Must haves:
- dialogue for the topic
- unlocks the `pastel_goth_outfit`

## Could haves:
- more dialogue unveiled with higher affinity
- With the new face slot we could add the face glitter for the `pastel_goth_outfit`, which was previously missing

## Is this feature request related to a problem? Please describe.
feature!

### PR DESCRIPTION
**Pull request by Blizzardsev**
Introduces the following:

- New topic `talk_fitting_clothing`, where Natsuki discusses her frustration with finding Nat-size clothes and how she works around it.
- Unlock method for the `jn_pastel_goth_getup` outfit, via `talk_fitting_clothing`.

Closes #517

### CODE REVIEW

**Comment by Blizzardsev** [comment]
@Obsidian-Knife Would love your input on this one!

File: game/script-topics.rpy [hunk 1]
```diff
@@ -9936,3 +9936,168 @@ label talk_daily_jokes_stop:
...
+    extend 3fcsbg " my imagination {i}never{/i} comes up short!"
+
+    $ pastel_goth_getup = jn_outfits.get_outfit("jn_pastel_goth_getup")
+    if not pastel_goth_getup.unlocked:
+        python:
+            import copy
+
+            pastel_goth_getup.unlock()
+            topic_outfit = copy.copy(pastel_goth_getup)
+            topic_outfit.hairstyle = jn_outfits.get_outfit(Natsuki.getOutfitName()).hairstyle
```
Comment (RaionArt) [inline game/script-topics.rpy:10022]: Worked without a problem for me!

**Review by RaionArt — APPROVED** [review]
It's always fun to see our little gremlin complaining about stuff! The flow of the topic is very nice and I'm glad to see the new eyes and eyebrows in action. The eyerolling fits perfectly to Natsuki!

File: game/script-topics.rpy [hunk 2]
```diff
@@ -9936,3 +9936,168 @@ label talk_daily_jokes_stop:
...
+    n 1csqsl "But {w=0.3}Oh.{w=0.75} My.{w=0.75} God,{w=0.75}{nw}"
+    extend 2csrem " [player]."
+    n 2fsqem "Clothes shopping.{w=1}{nw}"
+    extend 1fcsan " I swear.{w=1}{nw}"
+    extend 4fbkwr " It's like my own special hell!"
+    n 4fcswrl "Have you even {i}considered{/i} what getting new clothes is like when you're this small?!"
+    n 2fllfll "Seriously -{w=0.5}{nw}"
+    extend 2fbkwrl " it's the worst!{w=1}{nw}"
+    extend 2fcsgs " And it doesn't even matter where you go!"
+    n 4ftlfl "Medium?{w=0.5}{nw}"
```
[the remark this answers is not in the document]
Comment (Blizzardsev) [reply game/script-topics.rpy:9981]: Had to use them somewhere!

**Comment by Obsidian-Knife** [comment]
Agreed with Raion, nice topic!

**Pull request merged**